sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/_version.py

```diff
@@ -0,0 +1,7 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.0.6246, generator: {generator})
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
```

Comment:
I don't think you should generate with the version of the wrapped client. I think you should still overwrite the user agent and just not generate the generated code with a package version

Reply:
Yeah after we discussed this I made a note to create a separate user agent file